sklearn/sklearn-template/template/trainer/task.py

```diff
@@ -46,18 +103,41 @@ def _train_and_evaluate(estimator, dataset, output_dir):
 
   logging.info(scores)
 
-  # TODO(cezequiel): Write model and eval metrics to `output_dir`.
-  _ = output_dir
+  # The default name of the metric is training/hptuning/metric.
+  # We recommend that you assign a custom name. The only functional difference is that
+  # if you use a custom name, you must set the hyperparameterMetricTag value in the
+  # HyperparameterSpec object in your job request to match your chosen name.
+  hpt = hypertune.HyperTune()
+  hpt.report_hyperparameter_tuning_metric(
+    hyperparameter_metric_tag='my_metric_tag',
+    metric_value=np.mean(scores),
+    global_step=1000)
+
+  timestamp = str(int(time.time()))
+  trial_id = str(hpt.trial_id)
+
+  # Export to the folder of output_dir/trial_id/FILE_NAME_PREFIX_timestampFILE_NAME_SUFFIX
+  model_output_path = os.path.join(output_dir, trial_id,
```

Comment:
Suggest using `'{}_{}{}'.format(...)` to form the strings.

Reply:
Modified as recommended.